src/openassetio-python/package/openassetio/constants.py

```diff
@@ -49,64 +44,4 @@
 ## called from multi-threaded C++).
 kField_EntityReferencesMatchPrefix = "entityReferencesMatchPrefix"
 
-# Files
-
-kField_FilePath = "path"
-kField_FileExtensions = "extensions"
-kField_FileIsEnumerated = "enumerated"
-
-# Hints
-
-kField_HintPath = "pathHint"
-kField_HintFilename = "filenameHint"
-kField_HintName = "nameHint"
-
-# Image data
-
-kField_PixelColorspace = "colorspace"
-kField_PixelAspectRatio = "aspectRatio"
-kField_PixelWidth = "width"
-kField_PixelHeight = "height"
-kField_PixelNumChannels = "numChannels"
-kField_PixelEncoding = "encoding"
-kField_PixelCompression = "compression"
-kField_PixelBitDepth = "bitDepth"
-
-# Task/Editorial
-
-kField_Status = "status"
-
-# Sequence
-
-kField_FrameStart = "startFrame"
-kField_FrameEnd = "endFrame"
-kField_FrameIn = "inFrame"
-kField_FrameOut = "outFrame"
-kField_FrameRate = "frameRate"
-kField_DropFrame = "dropFrame"
-kField_FieldDominance = "fieldDominance"
-
-kField_MediaFrameIn = "mediaInFrame"
-kField_MediaFrameOut = "mediaOutFrame"
-
-## @}
-
-
-## @name Setting Key Names
-## Name for keys in the manager settings dict, where applicable.
-## @{
-
-kSetting_ManagerIdentifier = "OpenAssetIO:managerIdentifier"
```

Comment:
[question] Is the idea to drop common `.settings()` keys entirely, or just these ones because they're not useful?

Reply:
This was a legacy from the old `Session` API that used to have a `getSettings` mechanism, that actually added the current manager identifier into the settings dict itself, so that you could in theory store/reacall the whole thing in a host. We binned all that off [ages ago](https://github.com/OpenAssetIO/OpenAssetIO/pull/525/files#diff-fc4ebba0bbc6ffa1aaac5024dcb536722e303c6c695f452372a7ccb474aa6411L192).